picard/ui/options/general.py

```diff
@@ -18,7 +18,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 from PyQt5 import QtCore
-from PyQt5.QtWidgets import QInputDialog
```

Comment:
`QImportDialog` still needs to be imported because it is used in the `login(self)` method.

Reply:
Misclick :D

Reply:
I'm actually surprised that wasn't caught with the automatic checks.  Then again, I would have expected codacy to have caught it and it seems the codacy checking didn't run.